content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
+  <h1>Bonjour, monde !</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Il affiche un titre de section mentionnant « Bonjour, monde ! » sur la page.
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Cliquez sur le lien ci-dessus pour ouvrir un éditeur en ligne. Vous êtes libres de faire quelques changements et de voir comment ils affectent l'affichage. La plupart des pages de ce guide auront des exemples modifiables comme celui-ci.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Comment lire ce guide {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Dans ce guide, nous examinerons les éléments constituants les applications React: éléments et composants. Une fois maîtrisés, vous pourrez créer des applications complexes à partir de petites pièces réutilisables.
 
->Tip
+>Astuce
 >
->This guide is designed for people who prefer **learning concepts step by step**. If you prefer to learn by doing, check out our [practical tutorial](/tutorial/tutorial.html). You might find this guide and the tutorial complementary to each other.
+>Ce guide est destiné aux personnes qui préfèrent **apprendre étape par étape**. Si vous préférez apprendre par la pratique, allez voir notre [tutoriel pratique](/tutorial/tutorial.html). Vous trouverez peut-être ce guide et le tutoriel complémentaire.
 
-This is the first chapter in a step-by-step guide about main React concepts. You can find a list of all its chapters in the navigation sidebar. If you're reading this from a mobile device, you can access the navigation by pressing the button in the bottom right corner of your screen.
+Ceci est le premier chapitre d'un guide étape par étape à propos des concepts principaux de React. Vous pouvez trouver une liste des chapitres dans la barre latérale de navigation. Si vous lisez ceci depuis un appareil mobile, vous pouvez accéder à la navigation en appuyant sur le bouton situé dans le coin en bas à droite de votre écran.
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Chacun des chapitres de ce guide sont construits sur les connaissances introduites dans les chapitres précédents. **Vous pouvez apprendre la plupart de React en lisant les chapitres du guide « Concepts Principaux » dans l'ordre où ils apparaissent dans la barre latérale.** Par exemple, [« Introduction à JSX »](/docs/introducing-jsx.html) est le chapitre suivant.
```

Comment:
```suggestion
Chacun des chapitres de ce guide s'appuie sur les connaissances introduites dans les chapitres précédents. **Vous pouvez apprendre l’essentiel de React en lisant les chapitres du guide « Fondamentaux » dans l'ordre où ils apparaissent dans la barre latérale.** Par exemple, le prochain chapitre s’intitule [« Introduction à JSX »](/docs/introducing-jsx.html).
```

- Regarder le titre VF de la section du guide (« Fondamentaux », tsk tsk. L'importance de faire une repasse visuelle locale avant l'envoi final de la PR)
- Allègement et fluidification.

Reply:
@setheal idem, pas traité ce retour ?

Reply:
@tdd pareil un joli "cannot be applied", j'ai ajouté les modifications à la main :)